platform/security/security-token-storage/token-storage-api/src/main/java/org/codice/ddf/security/token/storage/api/TokenInformation.java

```diff
@@ -0,0 +1,51 @@
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * <p>This is free software: you can redistribute it and/or modify it under the terms of the GNU
+ * Lesser General Public License as published by the Free Software Foundation, either version 3 of
+ * the License, or any later version.
+ *
+ * <p>This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+ * without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ * GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public
+ * License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ */
+package org.codice.ddf.security.token.storage.api;
+
+import java.util.Map;
+import java.util.Set;
+
+/** Storage for a user's information */
+public interface TokenInformation {
```

Comment:
❓ Should the class name reflect that this deals with OIDC / JWT tokens?

Reply:
Did it confuse you? I was hoping to keep the names short and since it stores access and refreshes tokens I don't think we would confuse it with SAML.